### ISSUE TITLE
"suspect" not showing for species dx on event details page

### ISSUE THREAD

**Issue by JChipault** [USGS-WiM/whispersservices#423]
Even though the suspect box is checked, the word "suspect" is not being appended to the species diagnosis on the event details page (although it is appended to the event diagnosis on both the home page and the event details page). See event 200700 on the live site.

![image](https://user-images.githubusercontent.com/49287076/99271109-6b3f2e80-27ec-11eb-9a4e-2ac6e794c04c.png)

**Comment by BlakeDraper** [comment]
@aaronstephenson the calculation of the `diagnosis_string` field was changed mistakenly for the locationspecies response. The API should append ' suspect' to the end of the diagnosis name string where `suspect: true`. In addition to Jenny's example above, check out the response for event 170891 on the test DB: https://whisperstest.wim.usgs.gov/api/eventdetails/170891?no_page

Note that the `eventdiagnoses` has maintained the correct behavior. 

An abridged response to event 170891 request is provided below, showing the problem:
```
 "locationspecies": [
        {
          "id": 298394,
          "species_string": "Red-billed Teal",
          "population_count": null,
          "sick_count": null,
          "dead_count": 1,
          "sick_count_estimated": null,
          "dead_count_estimated": null,
          "priority": 1,
          "captive": false,
          "event_location": 134241,
          "species": 1440,
          "age_bias": null,
          "sex_bias": null,
          "speciesdiagnoses": [
            {
              "id": 25969,
              "organizations_string": [],
              "diagnosis_string": "White-nose Syndrome",
              "basis_string": null,
              "cause_string": "Cause of Death",
              "suspect": true,
              "priority": 1,
              "tested_count": null,
              "diagnosis_count": null,
              "positive_count": null,
              "suspect_count": null,
              "pooled": false,
              "location_species": 298394,
              "diagnosis": 516,
              "cause": 1,
              "basis": null,
              "organizations": []
            },
            {
              "id": 25970,
              "organizations_string": [],
              "diagnosis_string": "Adenovirus Hemorrhagic Disease",
              "basis_string": null,
              "cause_string": "Cause of Sickness",
              "suspect": true,
              "priority": 2,
              "tested_count": null,
              "diagnosis_count": null,
              "positive_count": null,
              "suspect_count": null,
              "pooled": false,
              "location_species": 298394,
              "diagnosis": 534,
              "cause": 2,
              "basis": null,
              "organizations": []
            }
          ]
        },
        {
          "id": 298369,
          "species_string": "Unidentified Duck",
          "population_count": null,
          "sick_count": 1,
          "dead_count": null,
          "sick_count_estimated": null,
          "dead_count_estimated": null,
          "priority": 2,
          "captive": false,
          "event_location": 134241,
          "species": 177,
          "age_bias": null,
          "sex_bias": null,
          "speciesdiagnoses": []
        }
      ],
```

**Comment by aaronstephenson** [comment]
Fixed in 2.0.10 5fa2d90
